src/libraries/System.Private.CoreLib/src/System/IO/RandomAccess.Windows.cs

```diff
@@ -163,13 +155,7 @@ internal static unsafe void WriteAtOffset(SafeFileHandle handle, ReadOnlySpan<by
                 }
 
                 int errorCode = FileStreamHelpers.GetLastWin32ErrorAndDisposeHandleIfInvalid(handle);
-                switch (errorCode)
-                {
-                    case Interop.Errors.ERROR_NO_DATA: // EOF on a pipe
-                        return;
```

Comment:
This was erroneous?

Reply:
Yes, this error was telling us "the pipe got closed/disconnected" and we were simply ignoring it and reporting successful write.

Reply:
Was that error code possible before?  If yes, this is actually a breaking change, since it's now throwing an exception for something we previously silently ignored?

Reply:
> Was that error code possible before? 

Yes, but before .NET 6 it was possible to access pipes via `FileStream` only by providing the handle in the `ctor` (it was blocked for paths), so it was rarely used. You are right, that is a breaking change.

Reply:
https://github.com/dotnet/docs/issues/32224